src/ResourceManager/KeyVault/Commands.KeyVault/help/Set-AzureKeyVaultCertificatePolicy.md

```diff
@@ -1,4 +1,4 @@
-﻿---
```

Comment:
@maddieclayton same comment about this call being a [`PATCH`](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Generated/KeyVaultClient.cs#L9542), so we should change the verb to `Update`

Reply:
Same comment as above - there is no Add-Policy, and we don't want one. A Set creates the policy if missing, and overwrites it if existing.